src/nwb_benchmarks/core/_network_tracker.py

```diff
@@ -0,0 +1,56 @@
+"""Base class for implementing benchmarks for evaluating network performance metrics for streaming read of NWB files."""
```

Comment:
```suggestion
"""
Network tracker for capturing network traffic and performance metrics during the execution of particular
methods or code snippets.
"""
```

Reply:
I had to trim this one a little bit to avoid a E501 pydocstyle warning in the linter (docstrings on modules must fit on single line)

Hope you don't mind